pkg/config/profile.go

```diff
@@ -163,16 +163,6 @@ func (p *Profile) GetTerminalPOSDeviceID() string {
 	return ""
 }
 
-// GetInstalledPlugins returns a list of locally installed plugins.
-// This does not vary by profile
-func (p *Profile) GetInstalledPlugins() []string {
-	if err := viper.ReadInConfig(); err == nil {
-		return viper.GetStringSlice("installed_plugins")
-	}
-
-	return []string{}
-}
-
```

Comment:
this didn't really belong here in the first place, and moving it was crucial for us to write adequate test coverage for the new uninstall command